vent/io/devices/sensors.py

```diff
@@ -288,14 +321,14 @@ def reset(self):
         """ Extended to add device specific behavior: Asks the sensor to perform a soft reset. 80 ms soft reset time."""
         super().reset()
         self.write_device(0x2000)
-        time.sleep(.08)
+        time.sleep(.08)  # TODO: this should be an await

```

Comment:
agreed - what should be awaited?

Reply:
That's just a reminder to myself that I'd like to make some of these sensor sleeps and gets asynchronous (for the future)